2025/3/20/make-counter/make-counter.ts

```
@@ -0,0 +1,7 @@
+export default function makeCounter(initialValue: number): Function {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)